sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/TestEntry.kt

```diff
@@ -0,0 +1,21 @@
+package ch.admin.bag.covidcertificate.eval.euhealthcert
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import java.io.Serializable
+import java.util.*
+
+@JsonClass(generateAdapter = true)
+data class TestEntry(
+	@Json(name = "tg") val disease: String,
+	@Json(name = "tt") val type: String,
+	@Json(name = "nm") val naaTestName: String?,
+	@Json(name = "ma") val ratTestNameAndManufacturer: String?,
+	@Json(name = "sc") val timestampSample: Date,
+	@Json(name = "dr") val timestampResult: Date?,
```

Comment:
Why have these as `Date`s but not the dates in the VaccinationEntry and RecoveryEntry?

Reply:
Was already like this in the generated Java classes. I will keep it like this for the moment